src/Main/Main.cpp

```diff
@@ -93,9 +93,8 @@ OptLohnerForm_t      OPT__FLAG_LOHNER_FORM;
 OptCorrAfterSync_t   OPT__CORR_AFTER_ALL_SYNC;
 OptTimeStepLevel_t   OPT__DT_LEVEL;
 
-bool                 ConservedRefInitialized = false;
-double               ConservedRef[1+NCONREF_MAX+NCOMP_PASSIVE+1];
-
+bool                 ConRefInitialized = false;
+double               ConRef[1+NCONREF_MAX+NCOMP_PASSIVE]; // time + conserved variables + passive scalars
```

Comment:
I prefer including `NCOMP_PASSIVE` within `NCONREF_MAX` (i.e., `ConRef[1+NCONREF_MAX]`) for simplicity (or even just `ConRef[NCONREF_MAX]`. This will also make the newly added sanity check in `Aux_Check_Conservation()` more intuitive. Let's discuss it in person.